modules/nf-core/quilt/quilt/main.nf

```diff
@@ -0,0 +1,62 @@
+process QUILT_QUILT {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "bioconda::r-quilt=1.0.4"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/r-quilt:1.0.4--r43h06b5641_2':
+        'biocontainers/r-quilt:1.0.4--r43h06b5641_2' }"
+
+    input:
+    tuple val(meta), path(bams), path(bais), path(bamlist), val(chr), val(regionStart), val(regionEnd), val(buffer), val(nGen), path(reference_haplotype_file), path(reference_legend_file), path(genetic_map_file)
```

Comment:
Is it necessary to have the bams and bamlist argument ?
Wouldn't it be possible to create the bamlist/cramlist from the bams argument ?

Reply:
The argument required by the module is the bamlist only, but it won't work if you don't have the bams in the same directory. The solution I thought of for a future implementation (in the pipeline or subworkflow) is to create the bamlist from the bams (which I think Saul did for the testing of STITCH). But you need some sort of channel manipulation, which is more appropriately done in the subworkflow context.